.github/workflows/integration-tests.yml

```diff
@@ -51,6 +58,6 @@ jobs:
         if: ${{ always() }}
         uses: actions/upload-artifact@v2
         with:
-          name: ${{ matrix.database.name }}-debug.log.xz
+          name: ${{ matrix.database.type }}-debug.log.xz
```

Comment:
With this change, the debug log from MariaDB and MySQL are having the same filename now. I would advise to revert this.